src/store/singing.ts

```diff
@@ -293,8 +278,14 @@ if (window.AudioContext) {
   clipper.output.connect(audioContext.destination);
 }
 
+type PhraseData = {
+  blob?: Blob;
+  source?: Instrument | AudioPlayer; // ひとまずPhraseDataに持たせる
+  sequence?: Sequence; // ひとまずPhraseDataに持たせる
+};
```

Comment:
おそらくこのデータは、Vuex内で閉じていて他で使ってほしくない値というニュアンスなのかなと思いました！
ローカル変数の概念が無いのでそう分けられたのかなと。

となると、コメントでこの型の説明付けておくとニュアンス伝わりやすいかもです。こんな感じとか？
```suggestion
// Phraseのstore内用データ
type PhraseData = {
  blob?: Blob;
  source?: Instrument | AudioPlayer; // ひとまずPhraseDataに持たせる
  sequence?: Sequence; // ひとまずPhraseDataに持たせる
};
```